flixel/input/gamepad/XboxButtonID.hx

```diff
@@ -125,16 +125,16 @@ class XboxButtonID
 	/**
 	 * Button IDs
 	 */
-	public static inline var A:Int = 10;
-	public static inline var B:Int = 11;
-	public static inline var X:Int = 12;
-	public static inline var Y:Int = 13;
-	public static inline var LB:Int = 8;
-	public static inline var RB:Int = 9;
-	public static inline var BACK:Int = 5;
-	public static inline var START:Int = 4;
-	public static inline var LEFT_ANALOG:Int = 6;
-	public static inline var RIGHT_ANALOG:Int = 7;
+	public static inline var A:Int = 0;
+	public static inline var B:Int = 1;
+	public static inline var X:Int = 2;
+	public static inline var Y:Int = 3;
+	public static inline var LB:Int = 4;
+	public static inline var RB:Int = 5;
+	public static inline var BACK:Int = 6;
+	public static inline var START:Int = 7;
+	public static inline var LEFT_ANALOG:Int = 8;
+	public static inline var RIGHT_ANALOG:Int = 9;
 	
 	public static inline var XBOX:Int = 14;
 	
```

Comment:
How would you use them with the FlxGamepad API though? gamepad.hat is a FlxPoint, and there already are getters for the dpad directions for which you don't need any indices.

Is 1. even an option? Does the hat event come with one of these indices?